test/e2eTests/tests/steps.d.ts

```diff
@@ -0,0 +1,623 @@
+
+type ICodeceptCallback = (i: CodeceptJS.I, Data:any, Apis:any, ClinicDashboardPage:any, ClinicPage:any, DashboardPage:any, EditPatientPage:any, LabDashboardPage:any, LabPage:any, LoginPage:any, PatientDetailPage:any, PharmacyDashboardPage:any, PharmacyPage:any, RegisterPatientPage:any, RegistrationDashboardPage:any, RegistrationPage:any, ReportsDashboardPage:any, SocialDashboardPage:any, SocialPage:any, VitalsAdultFormPage:any, VitalsDashboardPage:any, VitalsPage:any) => void;
```

Comment:
Love this!

Reply:
On second though this may be cumbersome to maintain. I was hoping to avoid having to make constant updates every time a new page is added and I'm not sure defining the name of the variable gets us anything. Something like this may work better:

type ICodeceptCallback = (i: CodeceptJS.I, ...args: any[])

If this codebase is ever moved to TS, and I hope it does, we can actually enforce types here. Until then I'm not sure enforcing `any` is very useful.